lib/pacemaker/pcmk_sched_graph.c

```diff
@@ -1257,7 +1257,7 @@ action2xml(pe_action_t * action, gboolean as_input, pe_working_set_t *data_set)
             hash2smartfield((gpointer)"pcmk_external_ip", (gpointer)value, (gpointer)args_xml);
         }
 
-        if (pe__is_guest_node(action->node)) {
+        if (action->node && pe__is_guest_node(action->node)) {
```

Comment:
I know but somehow clang wasn't smart enough to see that it would return NULL and thus not go into that section. So it needed that little piece of help here.
Things like that wouldn't go into the real fixes section - right?
I can split it into 2 lines and put a comment in.

Reply:
Right, not a real fix. Comment is probably a good idea otherwise someone will remove it later :)